Security/Test-ProxyLogon.ps1

```diff
@@ -181,17 +181,13 @@ process {
                 }
 
                 function Get-AgeInDays {
-                    param (
-                        [string]
-                        $dateString
-                    )
-
-                    $date = [DateTime]::MinValue
-                    if ([DateTime]::TryParse($dateString, [ref]$date)) {
-                        $age = [DateTime]::Now - $date
+                    param ( $dateString )
+                    if ( $dateString -and $dateString -as [DateTime] ){
+                        [System.Globalization.CultureInfo]$provider = [System.Globalization.CultureInfo]::InvariantCulture
```

Comment:
$provider is being flagged as an unused variable. Does this line do anything?

Also, can you run `.build\CodeFormatter.ps1 -Save` please? This will enforce the formatting rules and save the changes. If you run it a second time, it should then be clean, except for any PSAnalyzer errors.

Reply:
I took care of the formatting piece but I'm still not sure about this $provider.

Reply:
Sorry, the "[System.Globalization.CultureInfo]$provider = [System.Globalization.CultureInfo]::InvariantCulture" was left as part of trying to use the [DateTime]::TryParseExact  function.

Shall I generate a new pull request with this removed?

Reply:
Nah we can just update this one. I just wanted to check and make sure I wasn't missing something.